src/components/PostCard/PostCard.scss

```diff
@@ -8,11 +8,18 @@
   box-shadow: 0 0 8px #24272580;
   border-radius: 5px;
 
+  @media (width < 500px) {
+    width: 100%;
+    box-shadow: none;
+    border: 1px solid black;
+    padding: 8px;
+  }
+
   &__data {
     display: flex;
     gap: 8px;
 
-    @media (width < 400px) {
+    @media (width < 500px) {
```

Comment:
¿Por qué repetís el mismo breakpoint de responsive en tantos lugares?

Reply:
lo cambioamos
![IllDoMyBestCordellWalkerGIF](https://github.com/MarcosRodriguezAlonso/the-why-blog/assets/128187875/3517e2b9-4f35-4b59-b83e-800de46b5b33)